client/src/js/isTimezone.js

```diff
@@ -0,0 +1,8 @@
+const moment = require('moment-timezone');
+const timezones = moment.tz.names();
+
+module.exports = function(timezone) {
+	timezone = String(timezone);
```

Comment:
What can timezone be other than a string? Is it necessary to explicitly cast the value to a string?